src/shared/Footer/FooterUtils.ts

```diff
@@ -21,11 +19,11 @@ library.add(
 
 export interface SocialLink {
   url: string;
-  icon: any;
+  icon: typeof faTwitter;
```

Comment:
It's okay solution, but `typeof faTwitter` returns `IconProp` type which is also exported from Font Awesome library. It would be more readable